### PR TITLE
Update P2P hints

### PR DESCRIPTION
**Pull request by arthuro555**
- Remove the hint about using a custom broker as the default one is reliable and cloud hosted nowadays (it used to be often down)
- Allow closing the hint about data-loss
- Add a hint about P2P being a networking (not multiplayer) extension
- Add a hint about P2P leaking IP addresses
- Add a hint to use THNK 😉 😉 😉 😉

### CODE REVIEW

File: newIDE/app/src/Hints/index.js [hunk 1]
```diff
@@ -131,14 +132,17 @@ export const getExtraInstructionInformation = (type: string): ?Hint => {
...
-      message: t`Overriding the ID may have unwanted consequences. Do not use this feature unless you really know what you are doing.`,
+      message: t`Overriding the ID may have unwanted consequences, such as blocking the ability to connect to any peer. Do not use this feature unless you really know what you are doing.`,
     };
   }
   if (type.indexOf('P2P::') === 0) {
     return {
       kind: 'warning',
-      message: t`It is recommended to use your own custom broker server. Read the wiki page for more info.`,
-      identifier: 'p2p-broker-recommendation',
+      message: t`P2P is a peer-to-peer networking solution. It leaks IP addresses to other player - it is made to play with friends, and is not suitabale for playing with unknown players.
```
Comment (AlexandreSi) [inline newIDE/app/src/Hints/index.js:141]:
```suggestion
      message: t`P2P is a peer-to-peer networking solution. It leaks IP addresses to other player - it is made to play with friends, and is not suitable for playing with unknown players.
```

File: newIDE/app/src/MainFrame/Preferences/PreferencesContext.js [hunk 2]
```diff
@@ -147,8 +148,14 @@ export const allAlertMessages: Array<{
     label: <Trans>Lifecycle functions only included when extension used</Trans>,
   },
   {
-    key: 'p2p-broker-recommendation',
-    label: <Trans>Peer to peer broker server recommendation</Trans>,
+    key: 'p2p-is-networking',
+    label: (
+      <Trans>Peer to peer IP address leak warning/THNK recommendation</Trans>
```
Comment (AlexandreSi) [inline newIDE/app/src/MainFrame/Preferences/PreferencesContext.js:153]: Is it the text you want to display to users?

Comment (arthuro555) [reply newIDE/app/src/MainFrame/Preferences/PreferencesContext.js:153]: This is the toggle in the settings to disable/reenable the hint

Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/Preferences/PreferencesContext.js:153]: Oh yeah, I forgot about that!

**Pull request merged**